indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -53,17 +70,38 @@ export default function FormItem({
   // TODO move outside like with setupActions etc?
   const paidItemLocked = useSelector(state => isPaidItemLocked(state, rest.id));
   const isManagement = useSelector(getManagement);
+  const isUpdateMode = useSelector(getUpdateMode);
 
   const fieldRegistry = getFieldRegistry();
   const meta = fieldRegistry[inputType] || {};
   const InputComponent = meta.inputComponent;
   const inputProps = {title, description, isRequired, isEnabled, ...rest};
-  const disabled = !isEnabled || (paidItemLocked && !isManagement);
+  const showPurged = !setupMode && isPurged;
+  const disabled = !isEnabled || showPurged || (paidItemLocked && !isManagement);
+
+  let retentionPeriodIcon = null;
+  if (setupMode && !!retentionPeriod) {
```

Comment:
```suggestion
  if (setupMode && retentionPeriod) {
```

not needed here; the `!!` is only important when the falsy value might be used in a react render (where `0` would be rendered while `false` wouldn't)